src/data/faq_de.json

```
@@ -173,6 +173,18 @@
                             "Mithilfe dieser Schritt-für-Schritt Anleitung können Sie Zertifikate aus der Corona-Warn-App entfernen. Bitte beachten Sie, dass die ersten drei Schritte für sowohl iOS als auch Android gelten, aber nur jeweils einer der letzten beiden Schritte befolgt werden muss, je nachdem welches Betriebssystem mit Ihrem übereinstimmt. Impfzertifikate, Testzertifikate und Genesenenzertifikate werden als 'Zertifikate' zusammengefasst, da der Ablauf bei allen identisch ist.",
                             "<ul><li>Tippen Sie auf den Reiter 'Zertifikate'</li><li>Tippen Sie auf das Zertifikat, das Sie entfernen wollen</li><li>Scrollen Sie runter und tippen Sie auf die Kachel, die dem Zertifikat entspricht, das Sie löschen wollen</li><li>iOS: Tippen Sie unten auf dem Bildschirm auf 'Zertifikat entfernen'</li><li>Android: Tippen Sie auf das drei-Punkte-Icon oben rechts auf dem Bildschirm, und danach auf 'Entfernen'</li></ul>"
                         ]
+                    },
+                    {
+                        "title": "Warum erhalte ich die Fehlermeldung, dass mein Zertifikat ungültig ist?",
+                        "anchor": "hc_signature_invalid",
+                        "active": true,
+                        "textblock": [
+                            "Die Signatur des Zertifikats ist ungültig.",
+                            "Damit ist das ganze Zertifikat als digitales Dokument ungültig. Es kann daher nicht als Nachweis verwendet und auch nicht in die App importiert werden. Tritt diese Fehlermeldung bei einem bereits importierten Zertifikat auf, wird es nun als ungültig markiert.",
+                            "Prüfen Sie bitte, ob die Gültigkeit Ihres Zertifikats abgelaufen ist:",
+                            "<ul><li>In der Regel ist ein in Deutschland ausgestelltes Impfzertifikat ein Jahr ab Ausstellungdatum gültig.</li><li>Ein Genesenen-Zertifikat darf laut Festlegung der EU dagegen nicht länger als 180 Tage ab dem Datum des letzten positiven PCR-Tests gültig sein.</li></ul>",

```

```suggestion
                            "<ul><li>In der Regel ist ein in Deutschland ausgestelltes Impfzertifikat ein Jahr ab Ausstellungdatum gültig.</li><li>Ein Genesenenzertifikat darf laut Festlegung der EU dagegen nicht länger als 180 Tage ab dem Datum des letzten positiven PCR-Tests gültig sein.</li></ul>",
```
Spelling of "Genesenenzertifikat" as used in the app and elsewhere in the FAQs.

---

Changed it, thanks!